File: java/src/com/vittee/thainum/provider/DefaultUnitNameProvider.java
package com.vittee.thainum.provider;

public class DefaultUnitNameProvider extends BaseNameProvider implements UnitNameProvider {

    public DefaultUnitNameProvider() {
        super("", "สิบ", "ร้อย", "พัน", "หมื่น", "แสน", "ล้าน");
    }
}

File: csharp/ThaiNum/ThaiNum.Library/Provider/DefaultUnitNameProvider.cs
namespace ThaiNum.Library.Provider
{
    class DefaultUnitNameProvider : BaseNameProvider, IUnitNameProvider
    {
        public DefaultUnitNameProvider()
            : base("", "สิบ", "ร้อย", "พัน", "หมื่น", "แสน", "ล้าน")
        {

        }
    }
}
